pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -532,12 +606,58 @@ func (r *CloneReconcilerBase) syncCloneStatusPhase(syncState *dvSyncState, phase
 		event.eventType = corev1.EventTypeNormal
 		event.reason = CloneSucceeded
 		event.message = fmt.Sprintf(MessageCloneSucceeded, sourceNamespace, sourceName, dataVolume.Namespace, dataVolume.Name)
+	case cdiv1.CloneInProgress:
+		event.eventType = corev1.EventTypeNormal
+		event.reason = CloneInProgress
+		event.message = fmt.Sprintf(MessageCloneInProgress, sourceNamespace, sourceName, dataVolume.Namespace, dataVolume.Name)
+	case cdiv1.PrepClaimInProgress:
+		event.eventType = corev1.EventTypeNormal
+		event.reason = PrepClaimInProgress
+		event.message = fmt.Sprintf(MessagePrepClaimInProgress, dataVolume.Namespace, dataVolume.Name)
+	case cdiv1.RebindInProgress:
+		event.eventType = corev1.EventTypeNormal
+		event.reason = RebindInProgress
+		event.message = fmt.Sprintf(MessageRebindInProgress, dataVolume.Namespace, dataVolume.Name)
+	default:
+		r.log.V(1).Info("No event set for phase", "phase", phase)
```

Comment:
yeah that is technically correct.  There is a small window when the pvc is first created.  Maybe it should be a 3.  Thing is it would be pretty bad if this is encountered later in the processing like say a new phase is added to the populator that DV controller doesn't know about

Reply:
right, so I think can treat empty string or "PhaseUnset" which is empty string as the cdiv1.CloneScheduled case? and then the default will catch anything else?

Reply:
Looks like already doing that:  https://github.com/kubevirt/containerized-data-importer/blob/7654ef1b975e9bda3ac5fb792c7c97c77fb55342/pkg/controller/datavolume/clone-controller-base.go#L627